apps/demo-primeng/src/styles.css

```
@@ -0,0 +1,198 @@
+    rgb(15 23 42 / 0.92)
+  );
+}
+
+.summary {
+  margin: 0.75rem 0 0;
+  padding: 1rem;
+  border-radius: 0.9rem;
+  background: rgb(255 255 255 / 0.06);
+  color: var(--demo-summary-ink);
+  font-family:
+    'SFMono-Regular', 'IBM Plex Mono', 'Cascadia Code', Menlo, Monaco, Consolas,
```

Stylelint (17.10.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)